examples/default.yaml

```diff
@@ -418,6 +418,11 @@ networks:
 # env:
 #   KEY: value
 
+# Defines variables used for customizing the functionality.
+# These variables can be referenced as {{.Param.Key}} in lima.yaml.
+# param:
```

Comment:
bike shedding: param vs params

Reply:
I was also unsure which to choose, but I decided on `param` to mirror the usage of `env` and because it results in a shorter reference notation.

Reply:
>  I decided on `param` to mirror the usage of `env` and because it results in a shorter reference notation.

I'm fine with `param`, but if you want to go for the shortest notation, then there is also `arg`, with the same length as `env`. There is precedence for using it in `Dockerfile`. But for reasons I can't explain I find `param` more fitting.